scripts/mock-merkle-proof.ts

```diff
@@ -0,0 +1,76 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import express from "express";
+import { Identity } from "@semaphore-protocol/identity";
+import { Member } from "@semaphore-protocol/group";
+import { bigintToHex, hexToBigint } from "bigint-conversion";
+import { MerkleProof } from "@zk-kit/protocols";
```

Comment:
`@zk-kit/protocols` is deprecated, you can use `import { MerkleProof } from "@zk-kit/incremental-merkle-tree"
` instead.